ivy/functional/backends/tensorflow/experimental/activations.py

```diff
@@ -38,8 +38,8 @@ def thresholded_relu(
     return tf.cast(tf.where(x > threshold, x, 0), x.dtype)
 
 
-@with_unsupported_dtypes({"2.13.0 and below": ("complex",)}, backend_version)
-def relu6(x: Tensor, /, *, out: Optional[Tensor] = None) -> Tensor:
+# @with_unsupported_dtypes({"2.13.0 and below": ("complex",)}, backend_version)
```

Comment:
This line should probably be removed completely rather than just commented out

Reply:
Done